app/models/worldwide_organisation_edition.rb

```diff
@@ -0,0 +1,7 @@
+class WorldwideOrganisationEdition < Edition
+  include Edition::Organisations
+
+  def display_type_key
+    "worldwide_organisation_edition"
```

Comment:
I was really hoping this was used to make the name in the new document section, would have been an easy way around the type display issue